CesiumGltf/include/CesiumGltf/PropertyView.h

```diff
@@ -349,14 +408,20 @@ template <typename ElementType> class PropertyView<ElementType, false> {
       return;
     }
 
-    getNumericPropertyValues(classProperty);
+    if (classProperty.type != ClassProperty::Type::ENUM) {
+      getNumericPropertyValues(classProperty);
+    }
+
     if (_status != PropertyViewStatus::Valid) {
       return;
     }
 
     if (classProperty.noData) {
-      if (!_required) {
+      if (!_required && _propertyType == PropertyType::Enum &&
```

Comment:
Could we rationalize that if we get to this point and `propertyType == Enum` that `pEnumDefinition` is also non-null, so we no longer have to do a `nullptr` check? (Maybe an `assert` would help, or prove me wrong? 😝 )